python/src/scipp/_math.py

```diff
@@ -70,8 +69,7 @@ def reciprocal(x: DataArrayLike,
     return _call_cpp_func(_cpp.reciprocal, x, out=out)
 
 
-def sqrt(x: DataArrayLike,
-         out: Optional[DataArrayLike] = None) -> DataArrayLike:
+def sqrt(x: LabeledArray, out: Optional[LabeledArray] = None) -> LabeledArray:
```

Comment:
Should `out` args always require a keyword arg? I think so.

Reply:
Agree